### PR TITLE
feat: add '.columns-x' utility and friends classes

### PR DESCRIPTION
**Pull request by Techassi**
This adds the `.columns-x` and friends classes to be compatible with Tailwind v3.

#### Resources

- https://tailwindcss.com/docs/columns
- https://tailwindcss.com/docs/break-after
- https://tailwindcss.com/docs/break-before
- https://tailwindcss.com/docs/break-inside

#### Current status

- Utility class `.columns-x` added
- Classes `.break-after-x`, `.break-before-x` and `.break-inside-x` need to be added

#### Changes

This also adds the const `tShirtScale` (see [here](https://tailwindcss.com/docs/columns#adding-based-on-column-width)) which describes the sizes `xs - 7xl`. All previous occurences of these sizes were replaced by the new const `tShirtScale`.

Note: `.columns-x` adds 2 more sizes smaller than `xs`, `xs3` & `xs2`, which are out of scope of the "t-shirt" scale.

### CODE REVIEW

**Comment by Techassi** [comment]
Added `.break-after-x`, `.break-before-x` and `.break-inside-x` classes. Will now add coverage tests.

File: test/processor/resolve.test.ts [hunk 1]
```diff
@@ -44,8 +44,8 @@ describe('Resolve Tests', () => {
   });
 
   it('resolve static utilities', () => {
-    expect(Object.keys(processor.resolveStaticUtilities(false)).length).toEqual(300);
-    expect(Object.keys(processor.resolveStaticUtilities(true)).length).toEqual(309);
+    expect(Object.keys(processor.resolveStaticUtilities(false)).length).toEqual(313);
+    expect(Object.keys(processor.resolveStaticUtilities(true)).length).toEqual(322);
```
Comment (antfu) [inline test/processor/resolve.test.ts:48]:
```suggestion
    expect(Object.keys(processor.resolveStaticUtilities(false)).length).toEqual(320);
    expect(Object.keys(processor.resolveStaticUtilities(true)).length).toEqual(329);
```

**Comment by antfu** [comment]
@Techassi I could wait you a bit for the release if you have more to come :)

**Comment by Techassi** [comment]
@antfu Actually yes. I was preparing atleast one more PR. Will open later.

**Pull request merged**